app/assets/javascripts/darkswarm/services/cart.js.coffee

```diff
@@ -50,7 +50,7 @@ Darkswarm.factory 'Cart', (CurrentOrder, Variants, $timeout, $http, $modal, $roo
         @popQueue() if @update_enqueued
 
       .error (response, status)=>
-        @scheduleRetry(status)
+        RailsFlashLoader.loadFlash({error: t('js.cart.add_to_cart_failed')})
```

Comment:
I think we shoul keep the retry, the request can fail and we can retry. I'd keep the retry with a number, like 3 or 5.

Reply:
If we post a set of data to the cart and it definitely can't be processed, does it make sense to post exactly the same data again?

Here the user can still try to add or remove items after seeing the message, and it will send the new (different) data to the cart.

Reply:
agree with @Matt-Yorkley . It's not a unreliable integration where retrying would make sense, such as connecting to a third-party API, etc. IMO

Reply:
ok, if you guys agree, lets go.
I was thinking about the case: server is not available and cart becomes inconsistent with the order on the backend.